x-pack/test/functional/fixtures/kbn_archiver/canvas/lens.json

```diff
@@ -0,0 +1,151 @@
+        "indexpattern": {
+          "layers": {
+            "c61a8afb-a185-4fae-a064-fb3846f6c451": {
+              "columnOrder": [
+                "2cd09808-3915-49f4-b3b0-82767eba23f7"
+              ],
+              "columns": {
+                "2cd09808-3915-49f4-b3b0-82767eba23f7": {
+                  "dataType": "number",
+                  "isBucketed": false,
+                  "label": "Maximum of bytes",
+                  "operationType": "max",
+                  "scale": "ratio",
+                  "sourceField": "bytes"
+                }
+              }
+            }
+          }
+        }
+      },
+      "filters": [],
+      "query": {
+        "language": "kuery",
+        "query": ""
+      },
+      "visualization": {
+        "accessor": "2cd09808-3915-49f4-b3b0-82767eba23f7",
+        "layerId": "c61a8afb-a185-4fae-a064-fb3846f6c451"
+      }
+    },
+    "title": "Artistpreviouslyknownaslens",
+    "visualizationType": "lnsMetric"
+  },
+  "coreMigrationVersion": "8.0.0",
+  "id": "my-lens-vis",
+  "migrationVersion": {
+    "lens": "7.14.0"
+  },
+  "references": [
+    {
+      "id": "logstash-lens",
+      "name": "indexpattern-datasource-current-indexpattern",
+      "type": "index-pattern"
+    },
+    {
+      "id": "logstash-lens",
+      "name": "indexpattern-datasource-layer-c61a8afb-a185-4fae-a064-fb3846f6c451",
+      "type": "index-pattern"
+    }
+  ],
+  "type": "lens",
+  "updated_at": "2019-10-16T00:28:08.979Z",
+  "version": "WzYsMl0="
+}
+
+{
+  "attributes": {
+    "@created": "2018-11-19T19:17:12.646Z",
+    "@timestamp": "2018-11-19T19:36:28.499Z",
+    "assets": {},
+    "colors": [
+      "#37988d",
+      "#c19628",
+      "#b83c6f",
+      "#3f9939",
+      "#1785b0",
+      "#ca5f35",
+      "#45bdb0",
+      "#f2bc33",
+      "#e74b8b",
+      "#4fbf48",
+      "#1ea6dc",
+      "#fd7643",
+      "#72cec3",
+      "#f5cc5d",
+      "#ec77a8",
+      "#7acf74",
+      "#4cbce4",
+      "#fd986f",
+      "#a1ded7",
+      "#f8dd91",
+      "#f2a4c5",
+      "#a6dfa2",
+      "#86d2ed",
+      "#fdba9f",
+      "#000000",
+      "#444444",
+      "#777777",
+      "#BBBBBB",
+      "#FFFFFF",
+      "rgba(255,255,255,0)"
+    ],
+    "height": 920,
+    "isWriteable": true,
+    "name": "Test Workpad",
+    "page": 0,
+    "pages": [
+      {
+        "elements": [
+          {
+            "expression": "savedLens id=\"my-lens-vis\" timerange={timerange from=\"2014-01-01\" to=\"2018-01-01\"}",
+            "id": "element-8f64a10a-01f3-4a71-a682-5b627cbe4d0e",
+            "position": {
+              "angle": 0,
+              "height": 238,
+              "left": 33.5,
+              "top": 20,
+              "width": 338
+            }
+          }
+        ],
+        "id": "page-c38cd459-10fe-45f9-847b-2cbd7ec74319",
+        "style": {
+          "background": "#fff"
+        },
+        "transition": {}
+      }
+    ],
+    "width": 840
+  },
+  "coreMigrationVersion": "8.0.0",
```

Comment:
Another 8.0 export.

Reply:
Does that mean simply changing the value to 7.x, or rebasing off of the 7.x branch, or something else?

Reply:
https://github.com/elastic/kibana/pull/102533#discussion_r673150222

Reply:
Done, merging soon.